src/components/fieldset/index.md.njk

```diff
@@ -8,27 +8,27 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-{{ example({group: 'components', item: 'fieldset', example: 'default'}) }}
+{{ example({group: 'components', item: 'fieldset', example: 'default', html: true, nunjucks: true, open: false}) }}
 
 Use the Fieldset component to group related form inputs.
 
 ## When to use this component
 
 Use the Fieldset component when you need to show a relationship between multiple form inputs. For example, you may need to group a set of text inputs into a single fieldset when [asking for an address](../../patterns/addresses).
 
-{{ example({group: 'components', item: 'fieldset', example: 'address-group', html: true, nunjucks: true, open: true}) }}
+{{ example({group: 'components', item: 'fieldset', example: 'address-group', html: true, open: true}) }}
 
 If you’re using the examples or macros for [Radios](../radios), [Checkboxes](../checkboxes) or [Date input](../date-input), the fieldset will already be included.
 
 ## How it works
 
 The first element inside a fieldset must be a `legend` which describes the group of inputs. This could be a question, such as ‘What is your current address?’ or a statement like ‘Personal details’.
 
-{{ example({group: 'components', item: 'fieldset', example: 'default', html: true, nunjucks: true, open: false}) }}
+{{ example({group: 'components', item: 'fieldset', example: 'default', html: true, nunjucks: true, open: true}) }}
 
 On [Question pages](../../patterns/question-pages) containing a group of inputs, including the question as the legend helps users of screenreaders to understand that the inputs are all related to that question.
 
-Include any general help text which is important for filling in the form and can’t be written as [hint text](Link to hint text section of help users Fill in a form) in the legend, but try to keep it as concise as possible.
+Include any general help text which is important for filling in the form and can’t be written as [hint text](../../patterns/fill-in-a-form#hint-text) in the legend, but try to keep it as concise as possible.
```

Comment:
Is it possible to have a 'site route' variable instead of using relative urls here?

This link could break in the future otherwise.

Reply:
Noted, will merge now and we'll look into this.